### PR TITLE
Fix, improve input validation and add tests to ClickSend tts

### PR DESCRIPTION
**Pull request by CharlieBailly**
## Proposed change

- Update documentation of ClickSend TTS according to a PR in homeassistant core.
- Add a usage example.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76669
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/clicksend_tts.markdown [hunk 1]
```diff
@@ -13,42 +13,40 @@ ha_integration_type: integration
...
 notify:
   - platform: clicksend_tts
-    username: CLICKSEND_USERNAME
-    api_key: CLICKSEND_API_KEY
-    recipient: PHONE_NO
+    username: test@example.com
+    api_key: D83DED51-9E35-4D42-9BB9-0E34B7CA85AE
+    language: fr-fr
+    voice: male
+    recipient: +61411111111
```
Comment (frenck) [inline source/_integrations/clicksend_tts.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/clicksend_tts.markdown (hunk 1 above)
Comment (CharlieBailly) [reply source/_integrations/clicksend_tts.markdown]: Ok I will remove the optional requirements

File: source/_integrations/clicksend_tts.markdown [hunk 2]
```diff
@@ -59,4 +55,18 @@ voice:
...
 {% endconfiguration %}
 
-To use notifications, please see the [getting started with automation page](/getting-started/automation/).
+### Usage
+
+ClickSend is a notify platform and thus can be controlled by calling the notify service [as described here](/integrations/notify/). It will send a notification to the E.164 phone number you configured as **recipient**.
+
+```yaml
+alias: "The sun has set"
+description: ""
```
Comment (frenck) [inline source/_integrations/clicksend_tts.markdown]:
```suggestion
```

File: source/_integrations/clicksend_tts.markdown [hunk 3]
```diff
@@ -59,4 +55,18 @@ voice:
...
+trigger:
+  - platform: sun
+    event: sunset
+action:
+  - service: notify.clicksend_tts
+    data:
+      message: The sun has set
```
Comment (frenck) [inline source/_integrations/clicksend_tts.markdown]:
```suggestion
      message: "The sun has set"
```

File: source/_integrations/clicksend_tts.markdown [hunk 4]
```diff
@@ -13,42 +13,38 @@ ha_integration_type: integration
...
 
 ```yaml
 notify:
   - platform: clicksend_tts
-    username: CLICKSEND_USERNAME
-    api_key: CLICKSEND_API_KEY
-    recipient: PHONE_NO
+    username: test@example.com
+    api_key: D83DED51-9E35-4D42-9BB9-0E34B7CA85AE
+    recipient: +61411111111
```
Comment (frenck) [inline source/_integrations/clicksend_tts.markdown]: This should be changed back to the previous values, see also:

<https://developers.home-assistant.io/docs/documenting/standards#integration-and-platform-pages>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/clicksend_tts.markdown (hunk 4 above)
Comment (frenck) [reply source/_integrations/clicksend_tts.markdown]:
```suggestion
    username: CLICKSEND_USERNAME
    api_key: CLICKSEND_API_KEY
    recipient: PHONE_NO
```

**Review by frenck — APPROVED** [review]
Thanks, @CharlieBailly 👍

**Pull request merged**